lib/sdf-server/src/service/v2/change_set/approval_status.rs

```diff
@@ -25,10 +27,11 @@ pub async fn approval_status(
         .spicedb_client()
         .ok_or(ChangeSetAPIError::SpiceDBClientNotFound)?;
 
-    let calculator = ChangeSetApprovalCalculator::new(&ctx, spicedb_client).await?;
+    let (latest_approvals, requirements) =
+        dal_wrapper::change_set_approval::status(&ctx, spicedb_client).await?;
 
     Ok(Json(si_frontend_types::ChangeSetApprovals {
```

Comment:
In general, we shouldn't be returning JSON data, this should be a WsEvent.

Reply:
Event with a fetch route?

Reply:
🤦 i didn't realize it was GET, I thought this was the CRUD

Reply:
No worries LOL. Tbh this is a very side-effect-y thing that the commit description calls out, but I try not to include like this. Not your fault at all